packages/core/assistant-toolkit/src/initiative/util.ts

```
@@ -3,8 +3,9 @@
 //
 
 import * as Effect from 'effect/Effect';
+import * as Function from 'effect/Function';
```

Biome (2.3.14)

[error] Do not shadow the global "Function" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)